Turkey/Version.cs

```diff
@@ -15,6 +15,11 @@ public class Version: IComparable
 
         public static Version Parse(string input)
         {
+            if( string.IsNullOrEmpty(input) )
```

Comment:
Do you know where this code path might get triggered from?

Reply:
From the Tests :P (And that's about it...)

Reply:
Can we fix/modify the unit tests, then? I would rather have code fail ASAP if it runs into `null`s.

Reply:
Well null is valid value, null version means no SDK.

Reply:
Making it different non null value would require introducing some new properties somewhere else or something...

Reply:
Check this out, this needs to also happen to work with nulls: https://github.com/redhat-developer/dotnet-bunny/pull/27/commits/8dfaaa0a006a2296146b083255068aa1450ab253

Reply:
I still dont understand the problem. But given that multiple people are hitting it, we should hold back on a fix.